fedoragsearch-transforms/islandora_transforms/slurp_all_MODS_to_solr.xslt

```diff
@@ -43,7 +44,7 @@
     <xsl:variable name="family-n" select="mods:namePart[@type='family']"/>
     <xsl:variable name="t-o-address" select="mods:namePart[@type='termsOfAddress']"/>
 
-    <field name="utk_mods_etd_name_author_ms">
+    <field name="utk_mods_ir_name_author_ms">
```

Comment:
Switch field name to utk_mods_etd_name_author_ms

Reply:
Done